lib/adf_info.py

```diff
@@ -128,8 +140,24 @@ def __init__(self, config_file, debug=False):
             #Set data name to baseline case name:
             data_name = self.get_baseline_info('cam_case_name', required=True)
 
-        #End if
+            #Attempt to grab baseline start_years (not currently required):
+            syear_baseline = self.get_baseline_info('start_year')
+            eyear_baseline = self.get_baseline_info('end_year')
+
+            if syear_baseline and eyear_baseline is None:
```

Comment:
@nusbaume I'm still having some issues if I don't put parenthesis around the ```syear_baseline and eyear_baseline``` part of the if-statements here and on lines 179 and 189. The code skips over these without error without parenthesis, but it seems having the parenthesis was given pylint problems, correct?

Reply:
I ended up modifying this logic so that it checks for the start year and end year independently, so luckily this issue should no longer be a problem.